tests/tasks/execute_with_reboot.yml

```diff
@@ -0,0 +1,21 @@
+- name: Execute the kdump role and reboot if necessary
+  block:
+    - include_role:
+        name: linux-system-roles.kdump
+  rescue:
+    # Fail if failed for a different reason than kdump_reboot_required.
+    - name: handle errors
+      fail:
+        msg: "role failed"
+      when: not kdump_reboot_required
+
+    - name: restart managed host
+      shell: sleep 2 && shutdown -r now "Ansible updates triggered"
+      async: 1
+      poll: 0
+      ignore_errors: true
```

Comment:
you should use the reboot module: https://docs.ansible.com/ansible/2.9/modules/reboot_module.html

Reply:
I copy-pasted it from https://github.com/linux-system-roles/selinux/blob/master/tests/selinux_config_restore.yml#L17
Fixed now